docs/API.md

```diff
@@ -524,7 +530,7 @@ __Parameters__
 |`bucketName`  | _string_  |Name of the bucket |
 |`objectName` | _string_  |Name of the object  |
 |`filePath` | _string_  |Path to download object to |
-|`materials` | _encrypt.Materials_ | The module to decrypt the object data |
+|`materials` | _encrypt.Materials_ | Interface provided by `encrypt` package to encrypt a stream of data (For more information see https://godoc.org/github.com/minio/minio-go) |
```

Comment:
Need to update examples to show how to use with the encrypt option.

Reply:
AFAICS these are under FPutEncryptedObject()  and FGetEncryptedObject.